introduction.md

```diff
@@ -1,32 +1,44 @@
 # Introduction
 
-This operations-focused toolkit provides pragmatic, experience and
+This operations-focused toolkit provides pragmatic, experience- and
 evidence-based advice for implementing Digital Public Goods (DPGs) at
 the country level.  These best practices should be useful to any
 organization interested in creating a DPG -- including commercial
-companies -- but its main audiences are national government agencies
-and UN-affiliated agencies.  The toolkit was created through the
-support of the [UNICEF Office of
-Innovation](https://www.unicef.org/innovation/), as a co-champion of
-the [Digital Public Goods Alliance](https://digitalpublicgoods.net/).
+companies -- but their main audiences are decision-makers, project team leaders, 
+and project team members in national government and UN-affiliated 
+agencies.  It is a living document meant to be updated through public input as more 
+agencies develop implementation and management experience with DPGs. You can
+participate in its continued development and suggest changes at
+https://github.com/unicef/publicgoods-toolkit.    
 
 The toolkit takes a topic-focused approach, presenting practical
-recommendations around guidelines, tools, and frameworks that will
-help you create a successful DPG.  It can be read as a whole or per
-individual module and topic.  Topics include: understanding the types
-and value of open source communities and project lifecycles; national
-and agency level policies that relate to Free and Open Source Software
+recommendations around guidelines, tools, and frameworks to
```

Comment:
A lot of line breaks seemed to get added here, I'm not sure why.  It's not a problem, really; I'm just mentioning it in case it's a sign of some setting in your text editor or something that might want to be checked?

Reply:
Hmmm, I'm not seeing that .... I'm unsure what was going on. Sorry.